centos_ci.sh

```diff
@@ -292,50 +327,22 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
-function perform_docs_publish() {
-  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
-  REPO_OWNER=$(echo $REPO | cut -d"/" -f4)
-  cd gopath/src/github.com/minishift/minishift
-
-  git checkout master
-  git checkout -b docs-branch
-  git pull -f $REPO $BRANCH # Get changes from remote repo/branch
-  git log -n 1              # Display last commit in log as reference
-
-  install_docs_prerequisite_packages;
-  make gen_adoc_tar
-  build_openshift_origin_docs $(pwd)/docs/build/minishift-adoc.tar;
-
-  mkdir -p minishift/docs/ondemand/$REPO_OWNER-$BRANCH
-  cp -r openshift-docs/_preview/openshift-origin minishift/docs/ondemand/$REPO_OWNER-$BRANCH/ # Copy the openshift-origin
-  cp docs/build/minishift-adoc.tar minishift/docs/ondemand/$REPO_OWNER-$BRANCH/ # Copy doc tar
-  # http://stackoverflow.com/a/22908437/1120530; Using --relative as --rsync-path not working
-  RSYNC_PASSWORD=$RSYNC_PASSWORD rsync -av --delete --relative minishift/docs/ondemand/$REPO_OWNER-$BRANCH/ minishift@artifacts.ci.centos.org::minishift/
-  echo "Find docs tar here http://artifacts.ci.centos.org/minishift/minishift/docs/ondemand/$REPO_OWNER-$BRANCH/"
-  echo "Minishift documentation is hosted at http://artifacts.ci.centos.org/minishift/minishift/docs/ondemand/$REPO_OWNER-$BRANCH/openshift-origin/latest/minishift/index.html"
-}
-
-function build_and_test() {
-  setup_kvm_docker_machine_driver;
-  cd $GOPATH/src/github.com/minishift/minishift
+function perform_pr() {
   make prerelease synopsis_docs link_check_docs
-  # Run integration test with 'kvm' driver
   MINISHIFT_VM_DRIVER=kvm make integration_pr
-  echo "CICO: Tests ran successfully"
-  artifacts_upload_on_pr_and_master_trigger $1;
+  perform_artifacts_upload $1;
 }
 
-function prepare_for_proxy() {
-  export INTEGRATION_PROXY_CUSTOM_PORT=8181 #needs to be an unused port
-  firewall-cmd --zone=public --add-port=$INTEGRATION_PROXY_CUSTOM_PORT/tcp;
+function perform_master() {
```

Comment:
👍 Nice. Also nice imo that on master pushes we run all tests. This should give us early feedback. Do we have notifications via email configured for this job? If not, they should be.